win32/makeinit.ps1

```diff
@@ -1,4 +1,4 @@
-$mechanism = @("anonymous", "crammd5", "digestmd5", "scram", "gssapiv2", "kerberos4", "login", "ntlm", "otp", "passdss", "plain", "srp", "gs2")
+$mechanism = @("anonymous", "crammd5", "digestmd5", "scram", "gssapiv2", "kerberos4", "login", "otp", "passdss", "plain", "srp", "gs2")
```

Comment:
```suggestion
$mechanism = @("anonymous", "crammd5", "digestmd5", "scram", "gssapiv2", "login", "otp", "passdss", "plain", "srp", "gs2")
```

Reply:
Thanks, fixed in updated commit along with a few other missed references to the old kerberos4 plugin